autoscheduler/frontend/src/components/InfoPage/CaptionedGif/CaptionedGif.css

```diff
@@ -0,0 +1,40 @@
+video {
```

Comment:
Thanks for letting me know. Turns out I could just delete most of the duplicated CSS in the appropriate files without affecting the look of the page at all.  I think the way I left it makes sense because now the CSS for the captioned gif component and actual page content is separated but let me know if it should still be combined into one.